mcoding_bot/cogs/starboard.py

```diff
@@ -46,6 +46,11 @@ def embed_message(
         f"[Go to Message](https://discord.com/channels/"
         f"{bot.config.mcoding_server}/{msg.channel_id}/{msg.id})"
     )
+    if _obj_or_none(msg.attachments):
```

Comment:
isn't `MissingType` evaluated to `false` in boolean expr?

Reply:
@Sigmanificient _obj_or_none converts MissingType to None.